src/Azure.Functions.Sdk/Strings.resx

```diff
@@ -0,0 +1,177 @@
+    <data name="AZFW0101_Error_ExtensionPackageConflict" xml:space="preserve">
+        <value>The AzureFunctionsExtensionPackage {0}/{1} conflicts with existing package  {0}/{2}</value>
+    </data>
+    <data name="AZFW0102_Warning_ExtensionPackageDuplicate" xml:space="preserve">
+        <value>Duplicate extension package found: {0}/{1}</value>
+    </data>
+    <data name="AZFW0103_Error_InvalidExtensionPackageVersion" xml:space="preserve">
+        <value>The Invalid version for AzureFunctionsExtensionPackage '{0}': '{1}'. Ensure the Version metadata on this item is a valid NuGet package version.</value>
+    </data>
+    <data name="AZFW0104_Warning_EndOfLifeFunctionsVersion" xml:space="preserve">
+        <value>Azure Functions '{0}' is out of support and will not receive security updates in the future. Please refer to https://aka.ms/azure-functions-retired-versions for more information on the support policy.</value>
+    </data>
+    <data name="AZFW0105_Error_UsingLegacyFunctionsSdk" xml:space="preserve">
+        <value>Microsoft.NET.Sdk.Functions package is meant to be used with in-proc function apps. Please remove the reference to this package.</value>
+    </data>
+    <data name="AZFW0106_Error_UnknownFunctionsVersion" xml:space="preserve">
+        <value>The AzureFunctionsVersion '{0}' is unknown or not supported.</value>
+    </data>
+    <data name="AZFW0107_Error_UnsupportedTargetFramework" xml:space="preserve">
+        <value>Target framework '{0}' is not supported by Azure Functions. Minimum target framework is {1}</value>
+    </data>
+    <data name="AZFW0108_Error_CustomFunctionPackageReferencesNotAllowed" xml:space="preserve">
+        <value>Custom items in the '_AzureFunctionPackageReference' item group are not allowed.\n{0}</value>
+    </data>
+    <data name="Deploy_AsyncDeployment" xml:space="preserve">
+        <value>Asynchronous deployment. Polling status at {0}</value>
+    </data>
+    <data name="Deploy_BeginPublish" xml:space="preserve">
+        <value>Publishing {0} to {1}</value>
+    </data>
+    <data name="Deploy_CompletedFailure" xml:space="preserve">
+        <value>Failed to publish {0} to {1}.  Status: {2}</value>
+    </data>
+    <data name="Deploy_CompletedSuccess" xml:space="preserve">
+        <value>Successfully published {0} to {1}. Status: {2}</value>
+    </data>
+    <data name="Deploy_Failed" xml:space="preserve">
+        <value>Zip deployment to {0} has failed.\n    {1}\n    HTTP Status: {2}\n    Deployment Status: {3}</value>
+    </data>
+    <data name="Deploy_InvalidPublishUrl" xml:space="preserve">
+        <value>The publish url '{0}' is invalid. Publish url must be an absolute HTTP or HTTPS url.</value>
+    </data>
+    <data name="Deploy_PollFailure" xml:space="preserve">
+        <value>Failed to poll '{0}'. HTTP Status: {1}. Retry count {2}/{3}</value>
+    </data>
+    <data name="Deploy_Status" xml:space="preserve">
+        <value>Deployment status is {0}.</value>
```

Comment:
Do we have any deploy actions? How do these errors work? Is this supposed to work the with az cli/ core tools?